lib/internal/process/promises.js

```diff
@@ -99,3 +99,12 @@ function setupPromises(scheduleMicrotasks) {
 
   return emitPendingUnhandledRejections;
 }
+
+const savedString = String;
+function safeString(value) {
+  try {
+    return savedString(value);
+  } catch (err) {
+    return '<toString() threw exception>';
```

Comment:
What can a promise be rejected with that can't either be stringified in some way or `util/inspect`-ed?

I suppose if `toString()` throws we could either inspect the object or just `<Object>.prototype.toString()`?